src/common/constants/enums.ts

```diff
@@ -1,10 +1,10 @@
-export const roles = ['director_producer', 'director_of_photography']
+export const roles = ['director_producer', 'director_of_photography'] as const
 export type Role = (typeof roles)[number]
 export function roleTypeChecker(arg: any): arg is Role {
   return roles.includes(arg)
 }
 
-export const genres = ['documentary', 'animation', 'series']
+export const genres = ['documentary', 'animation', 'series'] as const
```

Comment:
Turns out this was why my compiler wasn't working.

Reply:
Super interesting

Reply:
Without the `as const`, typescript looks at that array and thinks it's an array of strings. We can change the values in the array during runtime without issue. However, `as const` tells typescript that these values never change. This is what allows us to use them as enums.